WTWSMS/common/landed_titles/mercenaries.txt

```diff
@@ -2565,4 +2565,91 @@ d_ghanan_band = {
 	strength_growth_per_century = 1.0
 	
 	mercenary_type = naval_merc_composition
+}
+
+c_mountain_jews = { # The Tzadikite Juhuro peoples ("Mountain Jews") had an immense and dedicated warrior culture. For the purposes of this Mod, the unique martial prowess takes the form of a special Mercenary Group under the Juhurian leader.
+
+	color = { 0.8 0.8 0.0 }
+	color2 = { 8 8 8 }
+
+	graphical_culture = easterngfx
+
+	capital = 676
+
+	allow = {
+		OR = {
+			culture_group = caucasian
+			culture = juhuro
+			religion = tzadikite
+			has_landed_title = c_kakheti
+			has_landed_title = d_iberia
+			has_landed_title = k_iberia
+		}
+	}
+
+	# Parent Religion 
+	religion = tzadikite
+	
+	culture = juhuro
+	
+	mercenary = yes
+
+	title = "WARCHIEF"
+	foa = "WARCHIEF_FOA"
+
+	# Always exists
+	landless = yes
+	
+	# Cannot be held as a secondary title
+	primary = yes
+
+	independent = no
+	
+	strength_growth_per_century = 0.1
+	
+	mercenary_type = juhuro_composition
+}
+
+c_trading_company = {
+	color = { 80 88 83 }
+	color2 = { 79 79 79 }
+	capital = 1369 #Socotra
+
+	graphical_culture = republicsgfx
+
+	# Hire Trigger
+	allow = {
+		capital_scope = {
+			OR = {
+				has_landed_title = c_socotra
+				has_landed_title = d_socotra
+				region = region_socotra
+				region = world_india_deccan
+				region = world_middle_east_arabia
+				region = world_africa_east
+			}
+			NOT = {
+				region = custom_diocese_aegyptus
+			}
+		}
+	}
```

Comment:
It's simple.

1. The Trading Company cannot be hired by the Egyptians, to ensure they don't spwan in the Meditteranean Sea, which is ahistorical.
2. Indians can hire them
3. Somalis and Ethiopians can hire them
4. Arabs can hire them
5. Persians can hire them

All of these locations match their RL scope of trade.